plugins/obs-qsv11/QSV_Encoder_Internal.h

```diff
@@ -54,6 +54,7 @@ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
 #pragma once
+#define ONEVPL_EXPERIMENTAL
```

Comment:
Right now in obs-deps, `BUILD_DISPATCHER_ONEVPL_EXPERIMENTAL` is set to `OFF`. Does that matter here with obs-studio as a consumer of VPL via obs-deps?

Reply:
It shouldn't need to be turned on. The flag can be kept off for now.

Reply:
Understood.